__tests__/server.test.js

```diff
@@ -315,4 +316,36 @@ describe('GET /api/users', () => {
             expect(errorMsg).toBe('404: Not Found')
         })
     })
+})
+describe('GET /api/articles?topic', () => {
+    test('200: responds with array of articles with matching topic', () => {
+        return request(app)
+        .get('/api/articles?topic=cats')
+        .expect(200)
+        .then(( { body } ) => {
+            const catArticles = body.articles
+            expect(Array.isArray(catArticles)).toBe(true)
+            expect(catArticles.length).toBe(1)
+            catArticles.forEach((catArticle) => {
+                expect(catArticle).toMatchObject({
+                    author: expect.any(String),
+                    title: expect.any(String),
+                    article_id: expect.any(Number),
+                    topic: expect.any(String),
```

Comment:
You should expect the topic to be cats if that is the query you have passed - proving you have the correct articles filtered